### PR TITLE
Update samples and snippets for the windows host to core beta6

### PR DESCRIPTION
**Pull request by andreasohlund**
All samples run fine

### CODE REVIEW

File: Snippets/Host/Host_7/Host_7.csproj [hunk 1]
```diff
@@ -58,4 +59,8 @@
     </Content>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
```
Comment (SzymonPobiega) [inline Snippets/Host/Host_7/Host_7.csproj]: Is this OK? I mean, why didn't we have that before?

File: Snippets/Host/Host_7/Host_7.csproj [hunk 2]
```diff
@@ -28,8 +28,9 @@
     <Reference Include="NServiceBus.Core">
       <HintPath>..\..\..\packages\NServiceBus.6.0.0-beta0006\lib\net452\NServiceBus.Core.dll</HintPath>
     </Reference>
-    <Reference Include="NServiceBus.Host">
-      <HintPath>..\..\..\packages\NServiceBus.Host.7.0.0-beta0003\lib\net452\NServiceBus.Host.exe</HintPath>
+    <Reference Include="NServiceBus.Host, Version=7.0.0.0, Culture=neutral, PublicKeyToken=9fc386479f8a226c, processorArchitecture=MSIL">
```
Comment (SzymonPobiega) [inline Snippets/Host/Host_7/Host_7.csproj]: What's the difference between full assembly name have and the short version previously?

Comment (andreasohlund) [inline Snippets/Host/Host_7/Host_7.csproj]: Not sure, I just did a nuget update?

File: Snippets/Host/Host_7/Host_7.csproj (hunk 1 above)
Comment (andreasohlund) [inline Snippets/Host/Host_7/Host_7.csproj]: damn init.ps1, will remove

Comment (andreasohlund) [inline Snippets/Host/Host_7/Host_7.csproj]: removed

File: Snippets/Host/Host_7/Host_7.csproj (hunk 2 above)
Comment (SzymonPobiega) [inline Snippets/Host/Host_7/Host_7.csproj]: @weralabaj do we care about this in reference files? I am new in the doco ;-)

Comment (SimonCropp) [inline Snippets/Host/Host_7/Host_7.csproj]: it depends on the version of nuget u use and how u do the update. both work fine

**Pull request merged**